packages/record-data/types/@ember/polyfills/index.d.ts

```diff
@@ -0,0 +1,7 @@
+/**
```

Comment:
Why do we have to define this? Doesn't DT have `@ember/polyfills`?

Reply:
it does but doesn't have the right signature, there was an open issue

Reply:
Let's make an issue for that and link here

Reply:
https://github.com/DefinitelyTyped/DefinitelyTyped/issues/38681